app/src/main/java/com/kickstarter/libs/rx/transformers/Transformers.java

```diff
@@ -184,4 +185,11 @@ private Transformers() {}
   public static @NonNull <T> ObserveForUITransformer<T> observeForUI() {
     return new ObserveForUITransformer<>();
   }
+
+  /**
+   * Emits the second of any input pair
+   */
+  public static <R> R second(final @NonNull Pair<?, R> anyPair) {
```

Comment:
Thanks for clarifying. I was worried that it didn't fit exactly in the Transformers class so this sounds good. I will move it and create the corresponding `first` method as well.